samples/web/content/apprtc/index.html

```diff
@@ -21,7 +21,19 @@
 </head>
 
 <body>
-
+  <script>
+    // Keep this in sync with the HTML element id attributes.
+    var uiConstants = {
```

Comment:
My intention was to keep the JS var close to the HTML to avoid out of sync
constants.
This seems impossible to Chrome app. So I'm going to move it to a JS file
and add comments to the HTML.

On Fri, Jan 9, 2015 at 1:01 PM, Chuck Hays <notifications@github.com> wrote:

> In samples/web/content/apprtc/index.html
> <https://github.com/GoogleChrome/webrtc/pull/316#discussion-diff-22742667>
> :
>
> > @@ -21,7 +21,19 @@
> >  </head>
> >
> >  <body>
> > -
> > +  <script>
> > +    // Keep this in sync with the HTML element id attributes.
> > +    var uiConstants = {
>
> Chrome app's can't have inline scripts, so this may need to be moved to a
> js file if we want to keep the html the same for both.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleChrome/webrtc/pull/316/files#r22742667>.
>



-- 
Jiayang Liu